### PR TITLE
Feature/Mongodb setup

### PR DESCRIPTION
**Pull request by slavlazar**
# Description

This is a simple mongo setup that uses [mongoose](https://mongoosejs.com/) as an ORM
You need to ensure you have [mongodb](https://www.mongodb.com/) set up for this to connect .. I suggest installing via homebrew
Set up a basic `User` schema

**Testing**

- [ ] Appropriate unit tests have been added

### CODE REVIEW

File: server/yarn.lock [hunk 1]
```diff
@@ -5505,6 +5613,13 @@ source-map@^0.7.3:
...
 
+sparse-bitfield@^3.0.3:
+  version "3.0.3"
+  resolved "https://registry.yarnpkg.com/sparse-bitfield/-/sparse-bitfield-3.0.3.tgz#ff4ae6e68656056ba4b3e792ab3334d38273ca11"
+  integrity sha1-/0rm5oZWBWuks+eSqzM004JzyhE=
+  dependencies:
+    memory-pager "^1.0.2"
+
 spdx-correct@^3.0.0:
   version "3.1.1"
```
Comment (vanvolkan) [inline server/yarn.lock:5624]: Considering we haven't really implemented much yet, the detail in package and yarn files are head spinning. :/

**Review by vanvolkan — APPROVED** [review]
Looks good.

File: server/yarn.lock (hunk 1 above)
Comment (slavlazar) [reply server/yarn.lock:5624]: Don't worry about the `.lock` files - these are just to ensure that all dependencies (and dependencies of our dependencies) use the exact same versions when installing the application - you can just ignore the `.lock` files when reviewing

Comment (vanvolkan) [reply server/yarn.lock:5624]: Can't ignore them when they cause merge conflicts :)

Comment (slavlazar) [reply server/yarn.lock:5624]: Yeah that was my bad - i initially set up dependencies that should've been in `devDependencies`

**Pull request merged**